scripts/bumpVersions.js

```diff
@@ -299,6 +310,26 @@ class VersionManager {
         this.addReleasedPackage(dep, bump);
       }
     }
+
+    // Bump anything that has this as a dep by a patch, all the way up the tree
+    if (status === 'released') {
```

Comment:
This block is pretty much exactly the same as the one above for non-released packages. Should we just combine them and do the same logic for all packages?